tdp/cli/commands/test_init.py

```diff
@@ -5,16 +5,17 @@
 from pathlib import Path
 
 from click.testing import CliRunner
+from sqlalchemy import Engine
 
 from tdp.cli.commands.init import init
 
 
-def test_tdp_init(collection_path: Path, database_dsn: str, vars: Path):
+def test_tdp_init(collection_path: Path, database_engine: Engine, vars: Path):
     args = [
         "--collection-path",
         collection_path,
         "--database-dsn",
-        database_dsn,
+        database_engine,
```

Comment:
I think, the arg should be a `str` no ?

Reply:
You're right, I mixed it with the `test_dag_order` tests. I fixed it, thanks!